packages/main/test/samples/ColorPalette.sample.html

```diff
@@ -42,109 +42,5 @@ <h3>Basic ColorPalette</h3>
 </xmp></pre>
 </section>
 
-<section>
```

Comment:
The remaining sample would now be Color Palette rather than Basic Color Palette

Reply:
Yeas I don't mind, will change it